### PR TITLE
chore: disable eta for headers & bodies

### PR DESCRIPTION
**Pull request by rkrasiuk**
## Motivation

Currently, ETA calculation is inconsistent and often ends up misleading the users.

Screenshot 2023-08-04 at 17 16 40

## Solution

Remove ETA from logs pending improvements.

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
agree, but would like to reenable after improvements,

for example there are cases where bodies is at 100% for a bit 

pending @shekhirin

**Comment by shekhirin** [comment]
I'd leave the ETA for execution stage, as it's the longest one and ETA can actually give a rough estimate on how long it'd take more to finish. Agree that for stages with more uneven commits it doesn't work good.

**Comment by joshieDo** [comment]
i think ETA is fine for stages that don't use the network (hashing, execution~~, etc)

**Comment by mattsse** [comment]
> I'd leave the ETA for execution stage, as it's the longest one and ETA can actually give a rough estimate on how long it'd take to finish. Agree that for stages with more uneven commits it doesn't work good.

good point, this actually outweighs the issues we have with it I think.

suggest we keep it as is and push improvements instead?

**Comment by shekhirin** [comment]
I still strongly agree with disabling it for headers/bodies stages 😄

File: bin/reth/src/node/events.rs [hunk 1]
```diff
@@ -299,6 +299,18 @@ impl Eta {
         self.last_checkpoint = current;
         self.last_checkpoint_time = Some(Instant::now());
     }
+
+    /// Format ETA for a given stage.
+    ///
+    /// NOTE: Currently ETA is disabled for Headers and Bodies stages until we find better
+    /// heuristics for calculation.
+    fn fmt_for_stage(&self, stage: StageId) -> String {
```
Comment (shekhirin) [inline bin/reth/src/node/events.rs:307]: should we instead completely remove the `eta` field if it's known to be `unknown` (lol) all the time?

**Review by mattsse — APPROVED** [review]

**Pull request merged**